tutor/templates/config/defaults.yml

```diff
@@ -40,7 +40,11 @@ LMS_HOST: "www.myopenedx.com"
 LOCAL_PROJECT_NAME: "{{ TUTOR_APP }}_local"
 MEILISEARCH_URL: "http://meilisearch:7700"
 MEILISEARCH_PUBLIC_URL: "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://meilisearch.{{ LMS_HOST }}"
-MEILISEARCH_INDEX_PREFIX: "openedx_{{K8S_NAMESPACE}}_"
+# Besides operators not using K8S, this variable is still defined as a default.
+# For operators running multiple installations, this default allows to support
+# multitenant meilisearch installations on K8S by default as installations are
+# deployed on different namespaces.
+MEILISEARCH_INDEX_PREFIX: "{{K8S_NAMESPACE}}_"
```

Comment:
How common a namespace change is?

Reply:
@Ian2012 Yeah, I actually would prefer that we let Tutor deploy a separate Meilisearch instance for each Open edX instance rather than having it centralized. And even if not, we can override this setting in our harmony plugin.